### PR TITLE
CIV-14235 api testing for GA

### PR DESCRIPTION
**Pull request by jeswanth-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-14235

### Change description ###

api testing for GA

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars */
+const config = require('../../config.js');
+const { assert } = require('chai');;
+let civilCaseReference;
+
+Feature('Create Lip v Lip claim -  Default Judgment');
```
Comment (pliao-hmcts) [inline e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js:6]: test tag is needed, @api-nightly or @api-cui ?

File: e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  console.log(civilCaseReference);
+  const { gaCaseState } = await api.createGAApplicationWithUnrepresented(config.applicantCitizenUser, civilCaseReference);
+  assert.equal(gaCaseState, 'AWAITING_APPLICATION_PAYMENT');
+})
+
+Scenario('Spec Claimant create GA with multiple application types', async ({ api }) => {
+  civilCaseReference = await api.createClaimWithUnrepresentedClaimant(config.applicantCitizenUser, 'SmallClaims', false, 'INDIVIDUAL');
+  const { gaCaseState } = await api.createGAApplicationWithUnrepresented(config.applicantCitizenUser, civilCaseReference, typeOfApplication = 'multiple');
+  assert.equal(gaCaseState, 'AWAITING_APPLICATION_PAYMENT');
+})
```
Comment (pliao-hmcts) [inline e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js:23]: AfterSuite is also needed

File: e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js (hunk 1 above)
Comment (pliao-hmcts) [reply e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js:6]: we will also need feature toggle condition to decide whether or not to run these tests on different env. in AAT we should not run this

File: e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js (hunk 2 above)
Comment (jeswanth-hmcts) [reply e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js]: added after suite has suggested

File: e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js (hunk 1 above)
Comment (jeswanth-hmcts) [reply e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js]: added tag api-cui

**Review by pliao-hmcts — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Review by manish14836 — APPROVED** [review]

**Pull request merged**